app/controllers/attachment_references_controller.rb

```diff
@@ -5,4 +5,13 @@ class AttachmentReferencesController < ApplicationController
   def show
     redirect_to @attachment_reference.url(filename: @attachment_reference.name)
   end
+
+  def destroy
+    if @attachment_reference.destroy
```

Comment:
Think we should `authroize(:edit, @attachment_reference.attachable)` when destroy. `show` is no need because the id is uuid.